### DISCUSSION TITLE
clip-path

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#271] [General]
I'v seen, that some top-solutions are solved with clip-path, but they begins width `clip-path:pat\68('m ...)`. I would like to transform my own clip-path-paths to this form, but I have no idea how to do it. Can someone explain it'
Thanks a lot!

**Comment by Beowolve** [comment]
Where do you create that path?

If you use:
http://yqnn.github.io/svg-path-editor/

You can just copy / paste that path from the top left into your code and it should work
